### PR TITLE
[Summer2021] Translate the registry_spi.md

### PR DESCRIPTION
**Pull request by QuakeWang**
According to this [document](https://dolphinscheduler.apache.org/zh-cn/docs/dev/user_doc/registry_spi.html), then translated it.

### CODE REVIEW

File: docs/en-us/dev/user_doc/registry_spi.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+#### How to use it？
+
+First you need to execute the `mvn -U install package -Prelease -Dmaven.test.skip=true` to install the plugin for generating the plugin JAR of the registry. The directory is: dolphinscheduler-dist/target/dolphinscheduler-dist-1.3.6-SNAPSHOT/lib/plugin/registry.
+
+Next, do the following configuration (using zookeeper as an example)
+
+|        Parameter        | Default                                                      | Description                              |
+| :---------------------: | ------------------------------------------------------------ | ---------------------------------------- |
+|    zookeeper.quorum     | localhost:2181                                               | zk Cluster connection information        |
```
Comment (CalvinKirs) [inline docs/en-us/dev/user_doc/registry_spi.md:11]: delete this. it's error config param

Comment (QuakeWang) [reply docs/en-us/dev/user_doc/registry_spi.md]: I have deleted it.

File: docs/en-us/dev/user_doc/registry_spi.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+#### How to use it？
+
+First you need to execute the `mvn -U install package -Prelease -Dmaven.test.skip=true` to install the plugin for generating the plugin JAR of the registry. The directory is: dolphinscheduler-dist/target/dolphinscheduler-dist-1.3.6-SNAPSHOT/lib/plugin/registry.
+
+Next, do the following configuration (using zookeeper as an example)
+
+|        Parameter        | Default                                                      | Description                              |
+| :---------------------: | ------------------------------------------------------------ | ---------------------------------------- |
+|   registry.plugin.dir   | /Users/username/workspaces/dolphinscheduler/dolphinscheduler-dist/target/dolphinscheduler-dist-1.3.6-SNAPSHOT/lib/plugin/registry | Registration Center Plugin Directory     |
+| registry.plugin.name\|  | zookeeper                                                    | Registration Center specific plugin name |
```
Comment (ptyp) [inline docs/en-us/dev/user_doc/registry_spi.md:12]: \ is redundant here.

Another thing is that only the first letters of the words in the first row of the Description column are capitalized. You should keep the consistency like changing the first letters of the remaining rows to upper case or changing the first letters of the first row to lower case.

File: docs/en-us/dev/user_doc/registry_spi.md [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+| registry.plugin.binding | registry                                                     | Dolphinscheduler plugin category         |
+|    registry.servers     | 127.0.0.1:2181                                               | ZK connection address                    |
+
+For the specific configuration information, please refer to the parameter information that provided by the specific plugin. Taking zk as an example, all the parameters of this configuration information about zk are in the class org/apache/dolphinscheduler/plugin/registry/zookeeper/ZookeeperConfiguration.java. If a parameter is needed to be changed to a specific value instead of the default, it can be configured directly in the registry. However, the prefix should be added to indicate this is a parameter of the registry. For example, "base.sleep.time.ms" should be configured in the registry as: "registry.base.sleep.time.ms=100".
+
+#### FAQ
+
+1: not found registry plugin
```
Comment (ptyp) [inline docs/en-us/dev/user_doc/registry_spi.md:20]: "registry plugin not found" would be more delicate.

File: docs/zh-cn/dev/user_doc/registry_spi.md [hunk 4]
```diff
@@ -20,7 +19,7 @@ registry.servers|127.0.0.1:2181|zk 连接地址
 #### FAQ
 1: not found registry plugin
```
Comment (ptyp) [inline docs/zh-cn/dev/user_doc/registry_spi.md:20]: If you change the English version, please remember to modify the Chinese version.

File: docs/en-us/dev/user_doc/registry_spi.md [hunk 5]
```diff
@@ -0,0 +1,26 @@
+### DolphinScheduler Registry SPI main design
+
+#### How to use it？
+
+First you need to execute the `mvn -U install package -Prelease -Dmaven.test.skip=true` to install the plugin for generating the plugin JAR of the registry. The directory is: dolphinscheduler-dist/target/dolphinscheduler-dist-1.3.6-SNAPSHOT/lib/plugin/registry.
+
+Next, do the following configuration (using zookeeper as an example)
```
Comment (ptyp) [inline docs/en-us/dev/user_doc/registry_spi.md:7]: "Next, please follow the configuration below (using zookeeper as an example)." should be better.

**Review by dailidong — APPROVED** [review]
+1
good job

**Pull request merged**